xdsl/dialects/riscv_scf.py

```diff
@@ -112,6 +132,60 @@ def verify_(self):
                         f"variables types."
                     )
 
+    def print(self, printer: Printer):
+        block = self.body.block
```

Comment:
another question is whether to factor this into one or multiple helpers since the scf and riscv_scf for loops are supposed to have consistent custom syntax

Reply:
I cannot forsee anything that would require different custom printing here, but up to you.